Assets/LeapMotion/Scripts/DataStructures/DeltaBuffer.cs

```diff
@@ -0,0 +1,152 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/// <summary> Allows you to add to a capped-size ring buffer of Vector3s and,
+/// when full, compute the buffer's average change over time. </summary>
+public class DeltaBuffer {
+
+  private RingBuffer<Vector3> _buffer;
```

Comment:
Looks like the code could be simplified if this was just a single buffer of an internal pair-struct instead of two buffers.